livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/Document.kt

```diff
@@ -1,37 +1,16 @@
 package org.livingdoc.repositories
 
-open class Document(val content: List<DocumentNode>)
-
-sealed class DocumentNode {
-    var result: Result = Result.Unknown
-    abstract val children: List<DocumentNode>
-
-    data class Text(
-            val text: String,
-            override val children: List<DocumentNode>
-    ) : DocumentNode()
-
-    data class TextList(
-            val items: List<Text>,
-            override val children: List<DocumentNode>
-    ) : DocumentNode()
-
-    data class DecisionTable(
-            val headers: List<String>,
-            val rows: List<DecisionTableRow>,
-            override val children: List<DocumentNode>
-    ) : DocumentNode()
-
-    data class DecisionTableRow(
-            val cells: Map<String, DecisionTableCell>,
-            override val children: List<DocumentNode>
-    ) : DocumentNode()
-
-    data class DecisionTableCell(
-            val text: String,
-            override val children: List<DocumentNode>
-    ) : DocumentNode()
-}
+open class Document(val tables: List<DecisionTable>, val lists: List<DecisionList>)
+
+abstract class ResultItem(val result: Result = Result.Unknown)
+
+data class DecisionTable(val headers: List<String>, val rows: List<DecisionTableRow>) : ResultItem()
+
+data class DecisionTableRow(val cells: Map<String, DecisionTableCell>)
+
+data class DecisionTableCell(val text: String) : ResultItem()
+
+data class DecisionList(val items: List<String>) : ResultItem()
```

Comment:
There is no such thing as a `DecisionList` bullet points / numbered lists are scenarios.

See als the [Glossary](https://github.com/LivingDoc/livingdoc/blob/master/doc/glossary.adoc)...

Reply:
done